### DISCUSSION TITLE
Microsoft Teams (new version) opens, but does not quit

### DISCUSSION THREAD

**Discussion by MartinHoefs** [ttscoff/bunch-docs#327] [General]
I've got the problem that Microsoft Teams (new version) opens without any problem via bunch, but no chance to quit it. 
All other apps in the same bunch perform without any problems, just MS Teams doesn't.
The older version of Teams opens and closes hassle-free, but not the new version.
All troubleshooting steps recommended in the bunch FAQ were executed, including using the filename and bundle identifier (com.microsoft.teams2), but nothing changed the behavior - also no success with forced quitting (!!!) using bundle identifier, real filename  from or standard filename ("Microsoft Teams"). Even the (quit everything) / (kill all) command doesn't seem to have any effect on the new MS Teams.

Any advice would be highly appreciated.

macOS 14.4.1 (23E224) on Intel Mac (iMac 15,1) and on Apple Silicon Mac (MacBook Pro M1)
bunch 1.4.11,166
Microsoft Teams (work or school) Version 24046.2813.2770.1094

**Comment by ukmac** [comment]
I am struggling with this one too, also seems to be the same issue with Outlook PWA

**Comment by MartinHoefs** [comment]
Looks like this issue has been resolved by Microsoft with the current Teams release (May 10th 2024, MS Teams Version 24091.213.2852.3731 )

**Comment by ukmac** [comment]
What are you using to in the bunch to close Teams ?

**Comment by MartinHoefs** [comment]
Hello Scott, with the app name it still seems to be troublesome, but when invoked via the bundle identifier
com.microsoft.teams2
it seems to work properly now. No forced closing necessary here - just open Teams with the bunch, when closing again it shuts down automatically (as the other apps do)

**Comment by ukmac** [comment]
AhI think I see the issue. I am trying to create a Bunch to close work apps that I have started manually. I will try opening Teams with Bunch and see if that helps
